<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>1001</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_INSERT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEARCH">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DELETE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ACTUALIZAR">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEARCH_RANGE">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH_RANGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputCodigo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputNombre">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputCarrera">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_INITIAL_DATA">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>INITIAL_DATA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
